clabverter/clabverter.go

```diff
@@ -210,15 +208,7 @@ func (c *Clabverter) load() error {
 		"determined fully qualified containerlab topology path as: %s", c.topologyPath,
 	)
 
-	// make sure we set working dir to the dir of the topo file, or the "parent" folder if its a url
-	switch {
-	case isURL(c.topologyFile):
-		pathParts := strings.Split(c.topologyFile, "/")
-
-		c.topologyPathParent = strings.Join(pathParts[:len(pathParts)-1], "/")
-	default:
-		c.topologyPathParent = filepath.Dir(c.topologyPath)
-	}
+	c.topologyPathParent, _ = filepath.Split(c.topologyFile)
```

Comment:
https://github.com/srl-labs/clabernetes/pull/44/files#diff-90f1180cd7621c5aca19753dec56c10a94a0c795922affb0e6a04d3f671d2e6dL213-R211

Im not 100% sure but I think this can be replace simlpy by filepath.Split(c.topologyFile) in main as well.